.github/pull_request_template.md

```diff
@@ -4,15 +4,10 @@ Thanks for contributing to Backpack :pray:
 Please include a description of the changes you are introducing and some screenshots if appropriate.
 -->
 
-+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-ios/blob/main/CONTRIBUTING.md)
-
```

Comment:
Why was this removed?

Reply:
Danger will now add a comment with this information instead for non-koala contributions

Reply:
I personally think the template is better for this as it will appear immediately and work with CLI tools, whereas danger will comment after several minutes 🤔

To be honest though I just `x` these when I make a PR anyway as I know the contribution process 🤷